explanations/stale-substitute.md

```diff
@@ -0,0 +1,5 @@
+The build log of this package indicates that `substituteInPlace` was called but failed to match its pattern.
```

Comment:
It [does not have to be just `substituteInPlace`](https://github.com/NixOS/nixpkgs/blob/9e0a3e5a645eca0fa383f5577ef12fbfa985a92d/pkgs/stdenv/generic/setup.sh#L673) by the way; although it is probably most common.